sklearn/ensemble/_hist_gradient_boosting/tests/test_loss.py

```diff
@@ -86,13 +86,14 @@ def fprime2(x):
 ])
 @pytest.mark.skipif(Y_DTYPE != np.float64,
                     reason='Need 64 bits float precision for numerical checks')
-def test_numerical_gradients(loss, n_classes, prediction_dim):
+@pytest.mark.parametrize('seed', range(1))
+def test_numerical_gradients(loss, n_classes, prediction_dim, seed):
```

Comment:
range(1) ?

Reply:
It's to make it easy to edit and put `range(100)` instead while only executing the test once by default.